client/x/evmstaking/types/withdraw.go

```diff
@@ -48,28 +44,3 @@ func NewWithdrawalFromMsg(msg *MsgAddWithdrawal) Withdrawal {
 		Amount:           msg.Withdrawal.Amount,
 	}
 }
-
-func MustMarshalWithdrawal(cdc codec.BinaryCodec, withdrawal *Withdrawal) []byte {
-	return cdc.MustMarshal(withdrawal)
-}
-
-// MustUnmarshalWithdrawal return the unmarshaled withdrawal from bytes.
-// Panics if fails.
-func MustUnmarshalWithdrawal(cdc codec.BinaryCodec, value []byte) Withdrawal {
-	withdrawal, err := UnmarshalWithdrawal(cdc, value)
-	if err != nil {
-		panic(err)
-	}
-
-	return withdrawal
-}
-
-// UnmarshalWithdrawal returns the withdrawal.
```

Comment:
We don't use this? Interesting, wonder why we have it in the first place

Reply:
Yeah, I did not find any usage in the code base.